config/lava/ltp/ltp-open-posix.jinja2

```diff
@@ -0,0 +1,15 @@
+- test:
+    name: {{ plan }}
+    timeout:
+      minutes: {{ job_timeout }}
+    definitions:
+       - repository: https://github.com/kernelci/test-definitions
+         from: git
+         revision: staging.kernelci.org
```

Comment:
Please use the `kernelci.org` branch in this PR, the staging branch should only be used on staging.